storage/storage.hpp

```diff
@@ -88,6 +88,10 @@ class Storage
   // downloading maps to a special place but not for continue working with them from this place.
   string m_dataDir;
 
+  // A list of urls of servers for downloading maps. It's necessary for storage integration tests.
+  // For example {"http://eu1.mapswithme.com/direct/mac/"}.
```

Comment:
It's not a big deal to remove it here. But what can we do in storage_integraion_tests? There's a string there: storage.SetDownloadingUrlsForTesting({"http://eu1.mapswithme.com/"});